mlflow/langchain/_langchain_autolog.py

```diff
@@ -253,7 +253,7 @@ def _inject_callbacks(original_callbacks, new_callbacks):
 
     def _is_new_callback_already_in_original_callbacks(new_callback, original_callbacks):
         return any(
-            isinstance(new_callback, type(original_callback))
+            isinstance(original_callback, type(new_callback))
```

Comment:
This seems the same as old ways, but should we check if the new_callback instances already exist in original_callbacks? cc @dbczumar